homeassistant/components/govee_ble/strings.json

```diff
@@ -17,5 +17,79 @@
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "event": {
+      "motion": {
+        "name": "Motion sensor",
```

Comment:
It can use device class translations

Reply:
f8b77bd1e9d1de87c4d9c7b0558d3debd8a47505